run_all_and_update_results.py

```diff
@@ -118,13 +119,19 @@ def main() -> None:
     )
     test_js_results = test_js_output["results"]["tests"]
 
+    concurrency = ""
+    runner_threads = os.environ.get("MAX_RUNNER_THREADS")
+    if runner_threads is not None:
+        concurrency = f"--concurrency {runner_threads * 2} "
```

Comment:
`runner_threads` will be a string, so while this works it doesn't give the intended result (just duplicates the string value). Let's just try parsing it as a base 10 int and ignore any errors:

```suggestion
    try:
        runner_threads = int(os.environ["MAX_RUNNER_THREADS"])
    except (KeyError, ValueError):
        concurrency = ""
    else:
        concurrency = f"--concurrency {runner_threads * 2} "
```

Reply:
Wow i didnt know `try: else:` was a thing, fixed btw :)